index.js

```diff
@@ -118,7 +123,7 @@ app.on('ready', () => {
         },
         {
           label: 'Quick Open (use this!)',
```

Comment:
the label could be dynamic, perhaps show what shortcut that has been set

Reply:
<img width="326" alt="CleanShot 2023-07-25 at 4 26 17@2x" src="https://github.com/smol-ai/menubar/assets/26971/ca377446-1079-426b-b9b1-409104be9a79">

My thinking was that the accelerator is dynamic and that would show what key is set. Open to doing whatever though.

Reply:
oh thats cool, all good then